### ISSUE TITLE
Is there any way to disable the zenity window popup when inserting a usb device?

### ISSUE THREAD

**Issue by GijsTimmers** [coldfix/udiskie#154]
Running udiskie 1.5.1 on ubuntu 17.10 with i3 window manager.

Whenever I insert a usb device, I get two notifications:
- a `notify-send` message: 
```
    Device added
    device appeared on /dev/sdc
```
- a `zenity` message:
```
    Device mounted
    /dev/sdc1: <USB-ID> mounted on /media/user/<USB-ID>
    [Browse] [Cancel] [OK]
```

My `config.yml`:

```
program_options:
    tray: false
    automount: true
    notify: true
    file_manager: ranger
```

I'm not interested in the zenity message and want to disable that one. I read https://github.com/coldfix/udiskie/blob/master/doc/udiskie.8.txt but could not find how to do what I want. How can I get this done?

**Comment by coldfix** [comment]
Hi,

the second one is a notify message as well. The zenity-style center-screen message is the result of the annoying way that your notification daemon handles notifications with actions. There are the following fixes:

- install a different notification daemon (and uninstall the first one). I recommend `xfce4-notifyd` that shows both messages as notifications

- disable actions in your config file:
```yaml
notification_actions:
 device_mounted: []
```

- disable the notification itself in the config file:
```yaml
notifications:
  device_mounted: false
```

I didn't check whether all of these options are available on 1.5.1, you are best served with upgrading to the latest version, 1.7.3!

**Comment by GijsTimmers** [comment]
Hi, thank you for explaining the problem. I uninstalled 1.5.1 and installed 1.7.3 with pip. 

The fixes work.
